nzediegwu1/EventManager
# #157533803 Secure Facility Component and refactoring

#### What does this PR do?
Secure Facility Component and refactoring

#### Description of Task to be completed?
1. Implement functionality to relectively render Facility component if current user is center owner
2. Refactor client code affected as follows:
   a. Include dist client and server folders to gitignore
   b. Include cloudinary API_KEY, UPLOAD_PRESET and API_SECRET in .env.sample file
   c. Install webpack modules for minimizing app bundle size
   d. Change start script to be optimized for production in package.json
   e. Include build and postinstall scripts in package.json
   f. Create separate webpack configs for dev and production which takes from webpack.common.config
   g. Comment out wrongly swagger setup in server/app.js

#### How should this be manually tested?
1. On terminal, run `npm run build && npm start`
2. On browser, login to the app and click centers in sidebar
3. Select a center owned by a different user, to navigate to centerDetails page
4. Observe that manageFacilities component and icon does not exist
5. Repeat step 3 with center created by you
6. Observe that the manageFacilities component was rendered!

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[#157533803](https://www.pivotaltracker.com/story/show/157533803)

#### Screenshots (If applicable)
N/A
